### PR TITLE
feat: Home page

### PR DESCRIPTION
**Pull request by dev-ashishsethi**
image
image
image

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,51 +1,82 @@
...
+}
+.profile-btn:focus div{
+  display: block;
+  position: absolute;
+  top: 3rem;
+}
+.profile-options>ul{
+  list-style-type: none;
+  text-align: left;
+  /* padding: 8px; */
```
Comment (PuneetP16) [inline src/App.css:52]: Suggestion: remove all the commented code and console.log before production

File: src/App.css [hunk 2]
```diff
@@ -1,51 +1,82 @@
...
-.links {
-  display: flex;
+.profile-options>ul>li:hover{
+  background-color: var(--red);
+  color: var(--white);
+  padding: 8px;
+}
+.sidebar {
+  grid-area: sidebar;
+}
```
Comment (PuneetP16) [inline src/App.css:70]: Suggestion: Format CSS properly.

**Review by PuneetP16 — APPROVED** [review]
Your UI looks great. check comments and good to merge.

File: src/Pages/SingleVideo/SingleVideo.jsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  const [singleVideo, setSingleVideo] = useState("");
+  const { customAxios } = useAxios();
+  const { video } = useVideo();
+  const [showDescription, setShowDescription] = useState(false);
+
+  const showMore=()=>setShowDescription(!showDescription);
+  
+
+  useEffect(() => {
+    (async () => {
```
Comment (ritikaawasthy) [inline src/Pages/SingleVideo/SingleVideo.jsx:22]: Suggestion: shifting these api calls to one file will make your code modular

File: src/Pages/SingleVideo/singleVideo.css [hunk 4]
```diff
@@ -0,0 +1,36 @@
+.video-page{
+    margin-top: 1.5rem;
```
Comment (ritikaawasthy) [inline src/Pages/SingleVideo/singleVideo.css:2]: good naming with css, its very clear

File: src/Utils/ViewsConvert.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+export function viewsConvert(views){
+    if (views>=1.0e+9) {
```
Comment (ritikaawasthy) [inline src/Utils/ViewsConvert.js:2]: why are views in 1.0e format?

File: src/Utils/categoryFilter.js [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+    const categoryList = response.categories
+      .filter((ele) => category.includes(ele.categoryName))
+      .map((ele) => ele.categoryId);
+
+    const videoList = videos.filter((video) =>
+      categoryList.includes(Number(video.snippet.categoryId))
+    );
+
+    console.log("category list", categoryList);
```
Comment (ritikaawasthy) [inline src/Utils/categoryFilter.js:18]: log statements will make your code render slow, so do remove them

**Review by ritikaawasthy — COMMENTED** [review]
UI looks really nice and your code is clean. Do check the comments

**Pull request merged**